lib/constable_web/templates/comment/edit.html.eex

```diff
@@ -1,5 +1,8 @@
 <div class="container">
   <div class="comment-edit">
+    <button class="tbds-button comment-preview-text-button">Write</button>
+    <button class="tbds-button comment-preview-markdown-button">Preview</button>
```

Comment:
What do you think about changing the color of these buttons? They seem too much like primary buttons (that bright red) when I think they're more secondary. 

![Screen Shot 2020-01-06 at 9 22 21 AM](https://user-images.githubusercontent.com/3245976/71823955-6bb45800-3066-11ea-970a-e371af9c4078.png)
![Screen Shot 2020-01-06 at 9 22 30 AM](https://user-images.githubusercontent.com/3245976/71823956-6bb45800-3066-11ea-8877-2f4adefc0aa3.png)

---

I added the `btn-secondary` class to them in a quick test, and now the problem is that their inactive state looks too much like it. 

![Screen Shot 2020-01-06 at 9 38 36 AM](https://user-images.githubusercontent.com/3245976/71824846-72dc6580-3068-11ea-9f6d-0c6b8d484a38.png)

---

Then I tried a couple of other background colors from he `_variables.scss` file: 

![Screen Shot 2020-01-06 at 9 38 58 AM](https://user-images.githubusercontent.com/3245976/71824865-7d96fa80-3068-11ea-92c0-7b4985109600.png)

![Screen Shot 2020-01-06 at 9 39 13 AM](https://user-images.githubusercontent.com/3245976/71824876-812a8180-3068-11ea-9166-0bd18feb649a.png)

---

I think I'm not the right person to figure this out 😄 but I do think it might make sense to not style them as the primary button (the "Post Comment" button). What do you think?

Reply:
Thanks for showing your thought process, that's really helpful. I believe you're correct that these buttons do not serve a primary purpose and perhaps should not sport the primary button styling.

I'll see what I can come up with, too, and maybe call upon a designer or two for their thoughts!

Reply:
Do you have any thoughts around this, @tysongach?

Reply:
A thought I have on the styling of these controls could involve
1. Subscribing the buttons to the `.tbds-button` + `.button-secondary` styles
2. Setting a border of zero on the inactive control

This would end up looking like:

![button-secondary-animation](https://user-images.githubusercontent.com/5240843/71920293-4cd3c580-313b-11ea-9904-bfd378f66f78.gif)

Reply:
@smharley was kind enough to lend a hand! This looks **great**. Thank you, Steven!

![image](https://user-images.githubusercontent.com/5240843/71936693-1445e300-315f-11ea-8a06-430677572292.png)